integration-tests/tests/mocked/nino-issue-credential/MockConfigFile.json

```diff
@@ -114,6 +125,42 @@
         }
       }
     },
+    "QuerySessionItemWithPersistentSessionId": {
```

Comment:
This is called `QuerySessionItemWithPersistentSessionId` but it doesn't appear to have a persistent session ID in it?

Reply:
This was a typo, should have been `QuerySessionItemWithoutPersistentSessionId`